### PR TITLE
New version: Ximalaya.Ximalaya version 3.2.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Successful
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | 喜马拉雅 | 喜马拉雅    |
| Version     | 3.2.0     | 3.2.0 |
| Publisher   | www.ximalaya.com   | www.ximalaya.com      |
| ProductCode |           | e2371ebd-7734-58dc-97bd-f6c9d997b351    |
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/88101)

### CODE REVIEW

File: manifests/x/Ximalaya/Ximalaya/3.2.0/Ximalaya.Ximalaya.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Tags:
+- audio
+- audiobook
+- ebook
+- podcast
+- radio
+- sound
+- voice
+# Agreements:
+ReleaseNotes: |-
```
Comment (Trenly) [inline manifests/x/Ximalaya/Ximalaya/3.2.0/Ximalaya.Ximalaya.locale.en-US.yaml:30]: This doesn't seem like en-US

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/x/Ximalaya/Ximalaya/3.2.0/Ximalaya.Ximalaya.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  - 评论图片支持放大缩小
```
Comment (stephengillie) [inline manifests/x/Ximalaya/Ximalaya/3.2.0/Ximalaya.Ximalaya.locale.en-US.yaml:31]:
```suggestion
  - Comment pictures support zoom in and zoom out.
```

File: manifests/x/Ximalaya/Ximalaya/3.2.0/Ximalaya.Ximalaya.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  - 音量支持设置到200
```
Comment (stephengillie) [inline manifests/x/Ximalaya/Ximalaya/3.2.0/Ximalaya.Ximalaya.locale.en-US.yaml:32]:
```suggestion
  - Supports setting volume to 200
```

File: manifests/x/Ximalaya/Ximalaya/3.2.0/Ximalaya.Ximalaya.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  - 专辑新增连载中和已完结标识
```
Comment (stephengillie) [inline manifests/x/Ximalaya/Ximalaya/3.2.0/Ximalaya.Ximalaya.locale.en-US.yaml:33]:
```suggestion
  - Added serialization and completed labels for albums.
```

File: manifests/x/Ximalaya/Ximalaya/3.2.0/Ximalaya.Ximalaya.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  - 更多功能内增加录音剪辑功能
```
Comment (stephengillie) [inline manifests/x/Ximalaya/Ximalaya/3.2.0/Ximalaya.Ximalaya.locale.en-US.yaml:34]:
```suggestion
  - Added more recording and editing functions.
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**